src/sources/opentelemetry/mod.rs

```diff
@@ -68,6 +68,7 @@ pub struct OpentelemetryConfig {
 #[configurable(metadata(docs::examples = "example_grpc_config()"))]
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
+#[configurable(metadata(docs::human_name = "gRPC"))]
```

Comment:
Should this be added to `WELL_KNOWN_REPLACEMENTS`?

Reply:
Oh yes, and actually, it's already there. I'll remove this addition.